client/src/styles/_fonts.scss

```diff
@@ -774,7 +765,202 @@
   content: "\e05d";
 }
 
-.font-icon-ticket:before {
```

Comment:
Has `.font-icon-ticket` been removed?

Reply:
Yes, it was accidentally added for a very short amount of time in this PR https://github.com/silverstripe/silverstripe-admin/pull/964. I doubt anyone is using it but I can add it back if it causes any concern.

Reply:
To be honest I don't like claiming that our icons are part of our API anyway. So it's fine by me